src/sqlittaaja/diff_check.py

```diff
@@ -0,0 +1,47 @@
+import difflib
+from typing import Any
+from zipfile import ZipFile
+from sqlittaaja.extractor import extract, student_info
+
+# Threshold percentage for warning >=
+threshold = 99
```

Comment:
Maybe we should get this from the config and default to some value if not present?